pubspec.lock

```diff
@@ -117,10 +117,10 @@ packages:
     dependency: transitive
```

Comment:
I think your pubspec is older. Probably need to update the branch. This will deal with `temp_frontend2.dart` as well